<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEdvid</class>
 <widget class="QWidget" name="FEdvid">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список единиц для выдачи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/table_go.png</normaloff>:/res/1/table_go.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Обновить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/Refresh.png</normaloff>:/res/Refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVid">
       <property name="text">
        <string>Выдать</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/saved_imports.png</normaloff>:/res/1/saved_imports.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Очистить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/cell_clear.png</normaloff>:/res/1/cell_clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/table_row_delete.png</normaloff>:/res/1/table_row_delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuit">
       <property name="text">
        <string>Выход</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/Exit.png</normaloff>:/res/Exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="pbVid">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
